src/main/java/uk/gov/ch/repository/psc/PersonsWithSignificantControlRepository.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.ch.repository.psc;
+
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import uk.gov.ch.model.psc.PersonWithSignificantControl;
+
+public interface PersonsWithSignificantControlRepository extends PagingAndSortingRepository<PersonWithSignificantControl, Long> {
+
+    @Query(value = "select " +
+            "CBA.CORPORATE_BODY_APPOINTMENT_ID," +
+            "CBA.OFFICER_FORENAME_1, " +
+            "CBA.OFFICER_FORENAME_2, " +
+            "CBA.OFFICER_SURNAME, " +
+            "CBA.APPOINTMENT_TYPE_ID, " +
+            "CBA.SERVICE_ADDRESS_LINE_1, " +
+            "CBA.SERVICE_ADDRESS_POST_CODE, " +
+            "CBA.SERVICE_ADDRESS_POST_TOWN, " +
+            "CBA.SUPER_SECURE_PSC_IND, " +
```

Comment:
This ill need changing, we are not supposed to be selecting the super secure officers, just the secure officers.
We should be capturing OD.SECURE_DIRECTOR_SERVICE_IND instead and add a where clause of CBA.SUPER_SECURE_PSC_IND = 'N'. There is reference to it in AC3 in https://companieshouse.atlassian.net/secure/RapidBoard.jspa?rapidView=186&projectKey=NCS&modal=detail&selectedIssue=NCS-71